### PR TITLE
Docs/schema change

### PR DESCRIPTION
**Pull request by PGijsbers**
## Change

Adds documentation on how to change the schema when adding an attribute to an existing entity.

## How to Test

Make sure you install the documentation dependencies (`pip install -e ".[docs]"` or refer to `pyproject.toml`).
Read and follow the documentation, see if it works, makes sense, and is intuitive.

## Checklist
- [x] Tests have been added or updated to reflect the changes, or their absence is explicitly explained.
- [x] Documentation has been added or updated to reflect the changes, or their absence is explicitly explained.
- [x] A self-review has been conducted checking:
  - No unintended changes have been committed.
  - The changes in isolation seem reasonable.
  - Anything that may be odd or unintuitive is provided with a GitHub comment explaining it (but consider if this should not be a code comment or in the documentation instead).
- [x] All CI checks pass before pinging a reviewer, or provide an explanation if they do not.

## Related Issues

See comment below.

### CODE REVIEW

**Comment by PGijsbers** [comment]
Some images for pages like the README currently do not load. This is because with the current organisation they could not render from the `docs/readme.md` page at the same time as from the `docs/developer/index.md` page. This would either require a separate post-processing plugin or to restructure the documentation. I will do this in a follow up PR. The documentation is currently not published anyway.

File: docs/developer/schema/attributes.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Working with Attributes
+In the metadata schema, every object has attributes that are represented by simple types like strings and numbers. 
+For example,`News` objects have a `source` and `Publication` objects have a `title`.
+This page details how to change attributes on existing metadata types.
+
+Here is an example of the code that defines the `start_time` attribute of the `Event` class:
```
Comment (Taniya-Das) [inline docs/developer/schema/attributes.md]: * `start_date` attribute

**Review by Taniya-Das — APPROVED** [review]
The documentation instructions are clear to me.

**Comment by PGijsbers** [comment]
@Taniya-Das Can I have an review looking at:
 - The new readme (docs landing page, docs/readme.md)
 - The new contributor guide (docs/contributing.md)
 - The trimmed old dev guide (docs/developers/index.md).. it's still in progress but i moved some stuff out and pics should work
 - Whether you think the current level of work-in-progress is acceptable for a merge. 
 Reason that I would want to merge is that I don't have the time to finish the rewrite this week, and others might be more inclined/find it easier to help this way.
 
 Other material wasn't changed (much) since last review. Thank you very much!

**Pull request merged**